sled-agent/src/services.rs

```diff
@@ -1423,17 +1421,23 @@ impl ServiceManager {
     fn zone_network_setup_install(
         gw_addr: &Ipv6Addr,
         zone: &InstalledZone,
-        static_addr: &Ipv6Addr,
+        static_addrs: &Vec<Ipv6Addr>,
```

Comment:
You should probably prefer to pass a slice rather than a vec here, that can save allocations in many of the callsites below.

e.g.:

```rust
static_addrs: &[Ipv6Addr],
```